app/javascript/components/public_health/PatientsTable.js

```diff
@@ -16,6 +16,8 @@ import {
   TabContent,
   Tooltip,
 } from 'react-bootstrap';
+
+import ReactTooltip from 'react-tooltip';
```

Comment:
Confused why this file is showing up in the diff as it this change is in `master`. Make sure you update this PR with the latest on `master`.